benchmark/benchmark_revisions.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Benchmarking tool for different commits
+"""
+# pylint: disable=subprocess-run-check
+import argparse
+import os
+import subprocess
+
+# ANSI escape sequences for terminal colors
+Colors = {
+    "red": "\033[31m",
+    "yellow": "\033[33m",
+    "blue": "\033[34m",
+    "magenta": "\033[95m",
+}
+RESET = "\033[0m"
+
+
+def col(text, color):
+    """Wraps the given text in color ANSI sequences.
+
+    Args:
+        text  (str): text to print
+        color (str): ANSI color code
+
+    Returns:
+        str: text wrapped with ANSI codes
+    """
+    return Colors[color] + text + RESET
+
+
+class cd:
+    """Context manager for changing the current working directory"""
+
+    def __init__(self, newPath):
+        self.newPath = os.path.expanduser(newPath)
+        self.savedPath = None
+
+    def __enter__(self):
+        self.savedPath = os.getcwd()
+        os.chdir(self.newPath)
+
+    def __exit__(self, etype, value, traceback):
+        os.chdir(self.savedPath)
+
+
+# benchmarking tool version
+__version__ = "0.1.0"
```

Comment:
Would this be a version specifically for the `revisions` or the entire `benchmark` module? Overall, a version could be included in a `_version.py` file in the `benchmark` directory (to follow PennyLane core).

Reply:
I copy-pasted this from the other files. I have no opinions on versioning, @smite, what did you have in mind there?

Reply:
I'd imagine that there is going to be a single version for the entire benchmarking suite